### PR TITLE
500 cigarettes

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request
Adds cigarettes to reqtorio.

## Why It's Good For The Game
Much needed recipe.
## Changelog
:cl:
add: Reqtorio can now produce 500 cigarettes.
/:cl:

### CODE REVIEW

**Comment by Ldip999** [comment]
I feel like it costs too much considering that you can get them for free at the vendors. Maybe they should be special cigarettes that could be sold for a profit simmilar to the pizza, or alternatively bestow some sort of benefit like chemrettes except not useless (Bicaridine kelotane tramadol tricord delight meralyne dermaline mix maybe?)
Also finding it goofy that they are made from metal.
`	name = "box of rounded metal plates"`

**Comment by usernameasterisk** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/blob/3bc96a22cbf452531a17f67f24fe78f3efd50d69/code/modules/reqs/supplypacks.dm#L1629-L1632

So 10 dollars = 1 credit.
You are paying 10 bucks for ONE cig, what a scam!

File: code/modules/factory/parts.dm [hunk 1]
```diff
@@ -879,14 +879,14 @@ GLOBAL_LIST_INIT(heavy_isg_recipe, list(
...
-	desc = "An incomplete FK-88 APFDS shell assembly."
-	result = /obj/item/ammo_magazine/heavy_isg/sabot
+
 
 /obj/item/factory_part/heavy_isg_sabot/Initialize(mapload)
 	. = ..()
 	recipe = GLOB.heavy_isg_recipe
+	name = "\improper FK-88 HE shell assembly"
+	desc = "An incomplete FK-88 APFDS shell assembly."
+	result = /obj/item/ammo_magazine/heavy_isg/sabot
```
Comment (QualityVan) [inline code/modules/factory/parts.dm:889]: What's up with this change?

Comment (Naaanii) [reply code/modules/factory/parts.dm:889]: Uhh, copy paste mistake I think.

**Comment by TiviPlus** [comment]
https://cdn.discordapp.com/attachments/504094319075131402/1256181263426125864/aSYQrCc.gif?ex=668126f7&is=667fd577&hm=657be31752732789bc9e934e6d7fbe6b6b441d489d133c4156857163d34b1d0b&

**Review by TiviPlus — APPROVED** [review]
This pr is brought to you by the tobacco lobby

**Pull request merged**